packages/main/src/components/delete-workspace/deleteWorkspace.js

```diff
@@ -0,0 +1,151 @@
+import axios from "axios";
+import { Link, useHistory } from "react-router-dom";
+import del from "./assets/delete.svg";
+import classes from "./deleteWorkspace.module.css";
+const DeleteWorkspace = () => {
+  const history = useHistory();
+  const orgName = localStorage.getItem("orgName");
+  const currentWorkspaceShort = localStorage.getItem("currentWorkspaceShort");
+  const currentWorkspace = localStorage.getItem("currentWorkspace");
+  const userData = JSON.parse(localStorage.getItem("userData"));
+  const token = userData.token;
+
+  const deleteWorkspace = async e => {
+    e.preventDefault();
+    const headers = {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${token}`
+    };
+    const res = await axios.delete(
+      `https://api.zuri.chat/organizations/${currentWorkspace}`,
```

Comment:
It's not good to reference the API you're calling here
use the base url constant provided in the app
it should be coming from @zuri-utilities

Reply:
I didnt't know the api that base_url belongs to so i have to make sure ap.zuri.chat is what I'm working with. Buh nevertheless i will change it

Reply:
It changes depending on the environment the app is being run in. So it's fine